src/server/common/templates/layouts/page.njk

```diff
@@ -9,6 +9,7 @@
 
 {% block head %}
   <link href="{{ getAssetPath('stylesheets/application.scss') }}" rel="stylesheet">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline' example.com; img-src 'self'"></meta>
```

Comment:
Hi @alexwbbr ~ you'll need to replace `example.com` with the domain that you want to allow src to run, for example if I want src code from google.com to be run then i'll add this